hack/make-rules/test-cmd-util.sh

```diff
@@ -3694,6 +3695,21 @@ __EOF__
     rm "${SAR_RESULT_FILE}"
   fi
 
+  # kubectl auth can-i
+  # kube-apiserver is started with authorization mode AlwaysAllow, so kubectl can-i always returns yes
```

Comment:
not sure how meaningful testing an authorization check is if the answer will always be yes